server/main.lua

```diff
@@ -38,18 +38,41 @@ QBX.UsableItems = {}
 ---@alias Model number
 ---@alias VehicleClass integer see https://docs.fivem.net/natives/?_0x29439776AAA00A62
 ---@type table<Model, VehicleClass>
-local vehicleClasses = {}
+local vehicleClasses = nil
+local vehicleClassesPromise = nil
```

Comment:
Aren't these already nil by default? Do they need to be explicitly defined?

Reply:
done.